packages/fx-core/templates/plugins/resource/apiconnector/sample/js/api-connector.template

```diff
@@ -0,0 +1,43 @@
+const teamsfxSdk = require("@microsoft/teamsfx");
+
+// Loads current app's configuration
+
```

Comment:
can you help follow the feature spec to remove the extra line breaks in source code?

Reply:
It is fixed. Thanks.